test/e2e/util.go

```diff
@@ -712,3 +724,72 @@ func getSigner(provider string) (ssh.Signer, error) {
 	}
 	return signer, nil
 }
+
+// Retrieve metrics information
+func getMetrics(c *client.Client) (string, error) {
```

Comment:
Once you sync please use this func in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/e2e/util.go#L727